config.go

```diff
@@ -124,6 +124,13 @@ type Config struct {
 	// Use of KeyLogWriter compromises security and should only be
 	// used for debugging.
 	KeyLogWriter io.Writer
+
+	// SessionStore is the container to store session for resumption.
+	SessionStore SessionStore
+
+	// FastResumption is the flag to allow fast resumption.
+	// In fast resumption, the server will not send HelloVerifyRequest msg.
+	FastResumption bool
```

Comment:
I'm not very sure `fast resumption` is a generally used term. (I couldn't find it in RFC and `crypto/tls`)
It would be better to use the word used in RFC or `crypto/tls`.

If there is any reference document, please put a comment to the code.

Reply:
Actually, there is no reference. I chosen this name personally. 

In the flight2 of handshake, the server will send the HelloVerifyRequest message to client, which will ensure the ClientHello come from a real address.

If the server be able to resume a previous session, there is no need to send the HelloVerifyRequest. As a result, I call this a FastResumption.

This process is only defined in dtls. So we cannot find any word in tls rfc or `crypto/tls`.

I am not a native English speaker. If you have a better name, feel free to let me know.